tests/src/coffee_httpd_integration_tests.rs

```diff
@@ -67,3 +67,241 @@ pub async fn httpd_init_add_remote() {
 
     cln.stop().await.unwrap();
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[ntest::timeout(560000)]
+pub async fn httpd_add_remove_plugins() {
+    init();
+
+    let mut cln = Node::tmp("regtest").await.unwrap();
+    let lightning_dir = cln.rpc().getinfo().unwrap().ligthning_dir;
+    let lightning_dir = lightning_dir.strip_suffix("/regtest").unwrap();
+    let manager = CoffeeHTTPDTesting::tmp(lightning_dir.to_string())
+        .await
+        .unwrap();
```

Comment:
please do not unwrap, but assert on the result if it is ok and then unwrap later (if the assert is false, print the debug version of the error

The rule here is that you should unwrap only when you know that the error never happens in the test can happen, so please remove the unwrap and fix all occurrence in the test

Reply:
Do you think this is a good use of `expect`?
We can
```rust
let manager = CoffeeHTTPDTesting::tmp(lightning_dir.to_string())
    .await
    .expect("Failed to start HTTPD");
```

Reply:
No, this is the same. We are missing information about the error

```rust
let manager = CoffeeHTTPDTesting::tmp(lightning_dir.to_string())
    .await;
assert!(manager.is_ok(), "{:?}", manager);
// SAFETY: This is safe to unwrap because we check that it is Ok in the step before.
let manager = manager.unwrap();
```

Every codebase has code-style rules, which are mine for the test. Panic inside the test makes it hard to read the error when something wrong happens quickly. So in the case of the assert the test is able to point you in the correct direction.

In other codebases could you find that the unwrap is allowed, but I would like to disallow this everywhere also inside the test.